<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>950</width>
    <height>720</height>
   </rect>
  </property>
  <property name="focusPolicy">
   <enum>Qt::FocusPolicy::StrongFocus</enum>
  </property>
  <property name="windowTitle">
   <string>How Long To Beat</string>
  </property>
  <property name="windowOpacity">
   <double>1.000000000000000</double>
  </property>
  <property name="styleSheet">
   <string notr="true">QMainWindow{
background-color: #1A1D23;
}

QPushButton {
background-color: rgb(97, 24, 45);
border-radius: 15px;
color: rgb(255, 255, 255);
min-height: 35px;
}

QPushButton:hover{
background-color: rgba(97, 24, 45, 200);
}
</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <widget class="QLabel" name="label">
      <property name="minimumSize">
       <size>
        <width>0</width>
        <height>30</height>
       </size>
      </property>
      <property name="font">
       <font>
        <family>8514oem</family>
        <pointsize>36</pointsize>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">color: white;</string>
      </property>
      <property name="text">
       <string>- How Long To Beat? -</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignmentFlag::AlignCenter</set>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLineEdit" name="txtSearchBox">
      <property name="minimumSize">
       <size>
        <width>0</width>
        <height>35</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>16777215</width>
        <height>16777215</height>
       </size>
      </property>
      <property name="focusPolicy">
       <enum>Qt::FocusPolicy::ClickFocus</enum>
      </property>
      <property name="styleSheet">
       <string notr="true">QLineEdit{
color: black;
background-color: rgba(255, 255, 255, 160);
border: none;
border-radius: 15px;
}</string>
      </property>
      <property name="text">
       <string/>
      </property>
      <property name="alignment">
       <set>Qt::AlignmentFlag::AlignCenter</set>
      </property>
      <property name="placeholderText">
       <string>Search game...</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QWidget" name="widget" native="true">
      <layout class="QHBoxLayout" name="horizontalLayout">
       <item>
        <widget class="QPushButton" name="searchToggle">
         <property name="minimumSize">
          <size>
           <width>0</width>
           <height>35</height>
          </size>
         </property>
         <property name="styleSheet">
          <string notr="true"/>
         </property>
         <property name="text">
          <string>Toggle Search Type</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="resetButton">
         <property name="cursor">
          <cursorShape>ArrowCursor</cursorShape>
         </property>
         <property name="text">
          <string>Reset Table</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
    <item>
     <widget class="QTableWidget" name="tableWidget_data">
      <property name="styleSheet">
       <string notr="true">QTableWidget {
    background-color: white;
    alternate-background-color: #EFCACA; 
	border-style: none;
}

QHeaderView::section {
    background-color: #2b2b2b;
    color: white;
    font-weight: bold;
    padding: 4px;
}


QScrollBar:vertical {
    background: transparent;
    width: 5px;
    margin: 0;
}

QScrollBar::handle:vertical {
    background: #555;
    min-height: 20px;
    border-radius: 10px;
}

QScrollBar::handle:vertical:hover {
    background: #333;
}

QScrollBar::add-line:vertical, QScrollBar::sub-line:vertical {
    height: 0;
    background: none;
    border: none;
}

QScrollBar::add-page:vertical, QScrollBar::sub-page:vertical {
    background: none;
}

</string>
      </property>
      <column>
       <property name="text">
        <string>Title</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>Completion</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>Genre(s)</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>Release</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>Platform(s)</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>Publisher(s)</string>
       </property>
      </column>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>950</width>
     <height>25</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
